### PR TITLE
Enabling FT for Mixtral

### PR DESCRIPTION
**Pull request by skanakamedal**
Added finetune_config.json in blob storage
image

Sample run -> https://ml.azure.com/experiments/id/8c49083b-2794-4416-91fa-6e2cda15120a/runs/0594d598-9062-495e-84ed-a82dc1f3b546?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&flight=itpmerge&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

### CODE REVIEW

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml [hunk 1]
```diff
@@ -20,8 +23,17 @@ tags:
       Standard_ND96asr_v4,
     ]
   inference_supported_envs:
-  - vllm
+    - vllm
+  finetune_compute_allow_list:
+    - Standard_ND40rs_v2
```
Comment (amangupta26) [inline assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:28]: what is the max sequence length supported for ND40? We can remove the compute if we dont have a validation on that part yet and add it incrementally

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml [hunk 2]
```diff
@@ -20,8 +23,17 @@ tags:
...
+    - Standard_ND96amsr_A100_v4
```
Comment (amangupta26) [inline assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:29]: Add other NDA100 SKU as well, check other model cards for it

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml (hunk 1 above)
Comment (skanakamedal) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:28]: The max supported is 32k but evaluation component might take time, so leaving it to -1 will pick 512 and user can increase it if necessary.

**Comment by amangupta26** [comment]
Did we test inference for finetuned model?

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml [hunk 3]
```diff
@@ -20,8 +23,17 @@ tags:
       Standard_ND96asr_v4,
     ]
   inference_supported_envs:
-  - vllm
+    - vllm
```
Comment (amangupta26) [inline assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:26]: Did we test inference for finetuned model?

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml (hunk 1 above)
Comment (amangupta26) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:28]: 512 is a small value. Let us keep a reasonable value of max sequence length in model specific defaults? 
What is user selects 32k as max sequence length, would it work on ND40?

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml (hunk 2 above)
Comment (skanakamedal) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:29]: I believe it is Standard_ND96asr_v4, added.

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml (hunk 1 above)
Comment (skanakamedal) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:28]: Yes 32 k will work on nd40.

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml (hunk 3 above)
Comment (skanakamedal) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:26]: I will test it on staging model, the older model MLmodel does not have base image.

File: assets/models/system/Mixtral-8x7B-v0-1/spec.yaml (hunk 1 above)
Comment (skanakamedal) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:28]: Shall I set context length to 2048? Also we know with 512 we got benchmarking on-par with mistral FYI.

Comment (skanakamedal) [reply assets/models/system/Mixtral-8x7B-v0-1/spec.yaml:28]: Setting to 2048 as dicussed.

**Pull request merged**